packages/js/product-editor/tsconfig-cjs.json

```diff
@@ -1,6 +1,11 @@
 {
 	"extends": "../tsconfig-cjs",
+	"include": [
+		"**/*",
+		"src/**/*.json"
+	],
 	"compilerOptions": {
-		"outDir": "build"
+		"outDir": "build",
+		"resolveJsonModule": true,
```

Comment:
Need to tweak TS configuration to support importing JSON files, as there are [benefits](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#benefits-using-the-metadata-file) to using the standard `block.json` file to provide metadata.

Reply:
+1